src/Configuration/Dotenv/Types.hs

```diff
@@ -0,0 +1,22 @@
+-- |
+-- Module      :  Configuration.Dotenv.Types
+-- Copyright   :  © 2015–2017 Stack Builders Inc.
+-- License     :  MIT
+--
+-- Maintainer  :  Stack Builders <hackage@stackbuilders.com>
+-- Stability   :  experimental
+-- Portability :  portable
+--
+-- Provides the types with extra options for loading a dotenv file.
+
+module Configuration.Dotenv.Types where
+
+-- | Alias for environment variables (NAME, VALUE).
+type NameValuePair = (String, String)
```

Comment:
I'd still propose to use bare `(String, String)` for reasons I mentioned earlier.